plat/rockchip/rk3399/drivers/dp/cdn_dp.c

```diff
@@ -41,3 +44,46 @@ __asm__(
 	".popsection\n"
 );
 
+static uint64_t *hdcp_key_pdata;
+static struct cdn_dp_hdcp_key_1x key;
+
+int hdcp_handler(struct cdn_dp_hdcp_key_1x *key);
+
+uint64_t dp_hdcp_ctrl(uint64_t type)
+{
+	switch (type) {
+	case HDCP_KEY_DATA_START_TRANSFER:
+		memset(&key, 0x00, sizeof(key));
+		hdcp_key_pdata = (uint64_t *)&key;
+		return 0;
+	case HDCP_KEY_DATA_START_DECRYPT:
+		if (hdcp_key_pdata == (uint64_t *)(&key + 1))
```

Comment:
+struct cdn_dp_hdcp_key_1x {
 +	uint8_t ksv[CDN_DP_HDCP_KSV_LEN];
 +	uint8_t device_key[CDN_DP_HDCP_DPK_LEN];
 +	uint8_t sha1[CDN_DP_HDCP_SHA_LEN];
 +	uint8_t uid[CDN_DP_HDCP_UID_LEN];
 +	uint16_t seed;
 +	uint8_t reserved[CDN_DP_HDCP_RESERVED_LEN];
 +} __aligned(8);

Maybe, can we should align the 8byte?